docs/src/quickstart.md

```diff
@@ -163,6 +163,10 @@ julia> JuMP.dual(con)
 -0.6
 ```
 
+!!! info
+    If you're confused about the sign associated with the dual value, read the
```

Comment:
Maybe `"See [constraint duality section](@ref constraint_duality) for a discussion of the convention that JuMP uses for signs of duals".`

It's not necessary to be confused in order to want to know about signs of duals.